static/js/containers/pages/DashboardPage.js

```diff
@@ -291,13 +274,7 @@ export class DashboardPage extends React.Component<
         </span>
       )
     }
-    if (isWithinEnrollmentPeriod(enrollment.run)) {
```

Comment:
I think the import of this function(`isWithinEnrollmentPeriod`) is now unused. Also, if this utility function is not used anywhere we can remove that whole function otherwise just remove the import from here.